rllib/algorithms/impala/impala_learner.py

```diff
@@ -50,25 +54,35 @@ class ImpalaLearner(Learner):
     def build(self) -> None:
         super().build()
 
-        # Build entropy coeff scheduling tools.
-        self.entropy_coeff_scheduler = Scheduler(
-            fixed_value=self.hps.entropy_coeff,
-            schedule=self.hps.entropy_coeff_schedule,
-            framework=self.framework,
-            device=self._device,
+        # Dict mapping module IDs to the respective entropy Scheduler instance.
+        self.entropy_coeff_schedulers_per_module: Dict[
+            ModuleID, Scheduler
+        ] = LambdaDefaultDict(
+            lambda module_id: Scheduler(
+                fixed_value_or_schedule=(
+                    self.hps.get_hps_for_module(module_id).entropy_coeff
+                ),
+                framework=self.framework,
+                device=self._device,
+            )
         )
 
+    @override(Learner)
```

Comment:
This was a bug: Forgot to do this, so right now, when removing a module, its entropy_coeff Schedule remains in memory.